src/ticpos.jl

```diff
@@ -0,0 +1,117 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _ticpos{T<:AbstractFloat}(deglen::T, pixlen::T, ticsize::T)
+    minpix = deglen*60/pixlen
+    incr = minpix*ticsize
+
+    if incr < 0
+        sgn = -1
+    else
+        sgn = 1
+    end
+
+    incr = abs(incr)
+
+    if incr >= 30
+        units = "Degrees"
+    elseif incr > 0.5
+        units = "Arc Minutes"
+    else
+        units = "Arc Seconds"
+    end
+
+    if incr >= 120
+        incr = 4
+    elseif incr >= 60
+        incr = 2
+    elseif incr >= 30
+        incr = 1
+    elseif incr > 15
+        incr = 30
+    elseif incr >= 10
+        incr = 15
+    elseif incr >= 5
+        incr = 10
+    elseif incr >= 2
+        incr = 5
+    elseif incr >= 1
+        incr = 2
+    elseif incr > 0.5
+        incr = 1
+    elseif incr >= 0.25
+        incr = 30
+    elseif incr >= 0.16
+        incr = 15
+    elseif incr >= 0.08
+        incr = 10
+    elseif incr >= 0.04
+        incr = 5
+    elseif incr >= 0.02
+        incr = 2
+    else
+        incr = 1
+    end
+
+    if units == "Degrees"
+        minpix = minpix/60
+    elseif units == "Arc Seconds"
+        minpix = minpix*60
+    end
+
+    ticsize = incr/abs(minpix)
```

Comment:
Right.  Indeed all argument look as  if they should be positive.  Maybe specify this in the docstring that the argument are assumed to be positive.